vm/systemSmartContracts/governance_test.go

```diff
@@ -1698,6 +1698,30 @@ func TestGovernanceContract_CloseProposalAfterGovernanceFixesShouldSetLastEndedN
 	require.Equal(t, uint64(6), lastEndedNonce)
 }
 
+func TestGovernanceContract_CannotClose(t *testing.T) {
```

Comment:
I guess this test only the backward compatibility flow. Otherwise, with the flag enabled, it should have returned true :)